rtoml/__init__.py

```diff
@@ -23,19 +23,19 @@ def load(toml: Union[str, Path, TextIO]) -> Dict[str, Any]:
     elif isinstance(toml, (TextIOBase, TextIO)):
         toml = toml.read()
 
-    return loads(toml)
+    return loads(toml, none_value=none_value)
 
 
-def loads(toml: str) -> Dict[str, Any]:
+def loads(toml: str, *, none_value: Optional[str] = None) -> Dict[str, Any]:
     """
     Parse a TOML string and return a python dict. (provided to match the interface of `json` and similar libraries)
     """
     if not isinstance(toml, str):
         raise TypeError(f'invalid toml input, must be str not {type(toml)}')
-    return _rtoml.deserialize(toml)
+    return _rtoml.deserialize(toml, none_value=none_value)
 
 
-def dumps(obj: Any, *, pretty: bool = False) -> str:
+def dumps(obj: Any, *, pretty: bool = False, none_value: str = "null", omit_none: bool = False) -> str:
```

Comment:
```suggestion
def dumps(obj: Any, *, pretty: bool = False, none_value: Optional[str, None] = "null") -> str:
```

I'm fine with adding omit functionality here, but at least in the python interface, it should be defined via the same parameter.

If you think `none_value=None` to mean "omit `None` values" is not clean enough, I'd be fine with a sentinel value, like:

```py
class OmitType:
    pass

Omit = OmitType()

... none_value: Union[str, OmitType] = 'null'
```

Reply:
No, I totally agree using `none_value=None` to omit `None` values.